test/RegisteringTest.t.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity ^0.8.13;
+
+import "./ZeroState.t.sol";
+
+contract RegisteringTests is ZeroState {
+
+    function testRegister() public {
+        console.log("registers a name");
+        vm.prank(sonic);
+        r.registerName("Sonic");
+        assertEq(r.checkName("Sonic"), sonic);
+        /*
+        had tried using
+        assertEq(r.nameToUser("Sonic"), sonic)
```

Comment:
that is so weird.  you shouldnt have had to create this getter.  i wonder if its something to do with assertEq ? Try:

```
require(r.nameToUser("Sonic"),  sonic);
```
or if that doesnt work another thing is sometimes Solidity chokes on string literals...

Don't spend a ton of time on this, if it doesnt work then reach out in the Slack channel and maybe someone can spot the problem

Reply:
Figured it out, I must of been using r.nameToUser["Sonic"]. Using r.nameToUser("Sonic") works as expected. So I subconsciously fixed the mistake when writing the comment???